toucan_connectors/__init__.py

```diff
@@ -55,6 +55,11 @@
         'label': 'Facebook Insights',
         'logo': 'facebook_insights/facebook-insights.png',
     },
+    'facebook_ads': {
+        'connector': 'facebookads.facebookads_connector.FacebookadsConnector',
+        'label': 'Facebook Ads',
+        'logo': 'facebookads/facebook_logo.png',
+    },
```

Comment:
cf commentaire sur le nommage + haut : 
```suggestion
    'FacebookAds': {
        'connector': 'facebook_ads.facebook_ads_connector.FacebookadsConnector',
        'label': 'Facebook Ads',
        'logo': 'facebook_ads/facebook-ads.png',
    },
```

Reply:
Renommé ici : https://github.com/ToucanToco/toucan-connectors/pull/349/commits/cd9695697a5154ed7d74ee70cc850cc473b1e026